### DISCUSSION TITLE
Replace images in multi-image TIF

### DISCUSSION THREAD

**Discussion by Jossie-X** [ImageMagick/ImageMagick#4648] [Digital Image Processing]
Hello,

my scans with SilverFast consist of TIFs with 3 images: high-resolution scan, low-resolution scan, IR-image. Along with the image are the tags in the metadata SilverFast needs to process the scan.

I want to manipulate the three images and then process the manipulated scans with SilverFast. So I have to ensure that all the metadata are maintained from the original. I tried to separate the images with imagemagick, work on them with imageJ and then re-combined them again with imagemagick. Finally I copied the tags with exiftool from the original to the manipulated scan file. But unfortunately that does not work. I suspect, that exiftool does not copy all the relevant tags.

So I wonder, if it is possible to copy the original scan first and then **_replace in this copy_** the three sub-images -- this way retaining the original metadata. Any ideas, how this could be done?

Many thanks an best wishes for the new year

Hermann-Josef

**Comment by snibgo** [comment]
> So I wonder, if it is possible to copy the original scan first and then replace in this copy the three sub-images -- this way retaining the original metadata. Any ideas, how this could be done?

Not with IM. IM can read the images from the input TIFF (ignoring some metadata), do any manipulations you want, and write a new TIFF.

I'm surprised that exiftool doesn't do what you want. Perhaps exif2 would be better.

Another option might be to use specialist TIFF tools. For example "tiffsplit" and "tiffcp", which are widely available.

**Comment by Jossie-X** [comment]
@snibgo 

Thanks a lot for your reply. I will give the other options a try. Perhaps I will also have to look into exiftool in more detail, what was lost on the way...

Best wishes for the New Year!

Hermann-Josef
